### ISSUE TITLE
Redis Integration

### ISSUE THREAD

**Issue by ducbot88** [o1lab/xmysql#57]
Hi Xmysql Team,

This is not an issue.

I’m using xmysql for my system. The table data is up to million rows. The performance will be slow day by day (still stable)

The best solution for faster speed is using redis (memcache).

Could you try to integrate xmysql with redis ? It will be the perfect solution working with mysql.

This is my desire to improve xmysql to be the best one.

Hope to hear from you.
Duc Hoang

**Comment by fanjindong** [comment]
interesting

**Comment by wiwengweng** [comment]
Yes, I also test 10000 concurrency on mysql with xmysql, which almost takes up to 200s with max response time(still alive).
